code/gbq/gbq_qr.py

```diff
@@ -80,13 +90,13 @@ def main():
     
     # load flu data
     fdl = FluDataLoader('../../data-raw')
-    df = fdl.load_data()
+    df = fdl.load_data(hhs_kwargs={'as_of': ref_date})
```

Comment:
we now request properly versioned data for runs for a past `ref_date`.

Reply:
Not related directly to this PR, but I noticed that the `load_data` function in `loader.py` has an empty dictionary (`hhs_kwargs`) in its signature.

Having mutable defaults in Python functions is considered a bad/dangerous practice (e.g. https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil).

So, more a note for future iterations, but would it be possible to use the `**kwargs` notation instead for `loader.py`?

Reply:
Fascinating!  I've filed a new issue for this, #13 .